src/main/java/io/ebeaninternal/server/rawsql/DRawSqlService.java

```diff
@@ -37,7 +37,7 @@ public RawSqlBuilder unparsed(String sql) {
   public SqlRow sqlRow(ResultSet resultSet, String dbTrueValue) throws SQLException {
     ResultSetMetaData meta = resultSet.getMetaData();
     int estCap = (int) (meta.getColumnCount() / 0.7f) + 1;
-    DefaultSqlRow ret = new DefaultSqlRow(estCap, 0.75f, dbTrueValue);
+    DefaultSqlRow ret = new DefaultSqlRow(estCap, 0.75f, dbTrueValue, false);
```

Comment:
Note to self ... this method takes binaryOptimizedUUID ... so we use that and not false here.